lib/block-supports/generated-classname.php

```diff
@@ -41,7 +41,10 @@ function gutenberg_get_block_default_classname( $block_name ) {
  * @return array Block CSS classes and inline styles.
  */
 function gutenberg_apply_generated_classname_support( $attributes, $block_attributes, $block_type ) {
-	$has_generated_classname_support = gutenberg_experimental_get( $block_type->supports, array( 'className' ), true );
+	$has_generated_classname_support = false;
```

Comment:
the default value is true here.

Reply:
OMG you're absolutely right... fixed :+1: